puppet/zulip_ops/templates/iptables/rules.v4.erb

```diff
@@ -32,14 +32,27 @@
 # FIXME: do something better here.
 -A INPUT -p udp -j ACCEPT
 <% else -%>
-# Accept incoming traffic on TCP ports 22 (SSH), 25 (SMTP), 80 (HTTP), 443 (HTTPS), and 5432 (Postgres)
+
+# Accept incoming traffic on TCP ports:
+#                       SSH
 -A INPUT -p tcp --dport   22 -j ACCEPT
```

Comment:
You can write the self-documenting `--dport ssh` here (using names in `/etc/services`).

Reply:
That does make it more readable -- thanks for the idea!